packages/@okta/vuepress-theme-prose/components/Home.vue

```diff
@@ -74,16 +20,133 @@
+              title: "Understand the basics of identity",
+              text: "Learn the key concepts you need for creating identity and access management (IAM) solutions for WIC.",
+              list: [
+                {
+                  url: "https://developer.okta.com/docs/concepts/iam-overview",
+                  text: "Understand IAM",
+                },
+                {
+                  url: "https://developer.okta.com/docs/concepts/how-okta-works",
+                  text: "How WIC works",
+                },
+                {
+                  url: "https://developer.okta.com/docs/concepts/oauth-openid",
+                  text: "Choose an authentication protocol",
+                },
+                {
+                  url: "https://developer.okta.com/signup",
+                  text: "Get a developer org",
+                },
+              ]
+            }
+          ]
+        },
+        {
+          title: "Build",
+          block: [
+            {
+              src: "/img/workforce2-light.svg",
+              src_dark: "/img/workforce2-dark.svg",
+              title: "Connect with APIs and SDKs",
+              text: "Build apps and services that interact directly with WIC for a completely integrated experience.",
+              list: [
+                {
+                  url: "https://developer.okta.com/docs/reference",
+                  text: "Explore reference APIs",
+                },
+                {
+                  url: "https://developer.okta.com/code",
+                  text: "Explore our SDKs ",
+                },
+                {
+                  url: "https://developer.okta.com/docs/guides/embedded-authentication",
+                  text: "Explore embedded authentication use cases",
+                },
+              ]
+            },
+            {
+              src: "/img/workforce3-light.svg",
+              src_dark: "/img/workforce3-dark.svg",
+              title: "Authenticate",
+              text: "Define how your applications and APIs verify the identity of a user or device.",
+              list: [
+                {
+                  url: "https://developer.okta.com/docs/guides/redirect-authentication",
+                  text: "Start with redirect authentication",
+                },
+                {
+                  url: "https://developer.okta.com/docs/guides/authenticators-overview/main",
+                  text: "Set up multifactor authentication",
+                },
+                {
+                  url: "https://developer.okta.com/docs/guides/identity-providers",
+                  text: "Use an external IdP"
+                }
+              ]
+            },
+            {
+              src: "/img/workforce4-light.svg",
+              src_dark: "/img/workforce4-dark.svg",
+              title: "Brand and customize",
+              text: "Tailor your IAM tools with your organization’s brand and give users a consistent, familiar experience",
```

Comment:
All the other items include a period. Either remove the period from those items, or add one here.